pytradfri/command.py

```diff
@@ -1,23 +1,28 @@
 """Command implementation."""
+from __future__ import annotations
 
 from copy import deepcopy
+from typing import Any, Optional, Callable
+
+TYPE_PROCESS_RESULT_CB = Optional[Callable[[Any], Optional[str]]]
+TYPE_ERR_CB = Optional[Callable[[str], str]]
 
 
 class Command(object):
     """The object for coap commands."""
 
     def __init__(
         self,
-        method,
-        path,
-        data=None,
+        method: str,
+        path: list[str],
+        data: Optional[dict[str, Any]] = None,
```

Comment:
Don't mix old and new style annotations.
```suggestion
        data: dict[str, Any] | None = None,
```

Reply:
I did not see it as old/new, but you are right they are all changed.